frontend/packages/ux-editor/src/components/TextResourceEdit.test.tsx

```diff
@@ -79,8 +80,31 @@ describe('TextResourceEdit', () => {
     await act(() => user.type(textBox, additionalValue));
     await act(() => user.tab());
     expect(queriesMock.upsertTextResources).toHaveBeenCalledTimes(1);
-    expect(queriesMock.upsertTextResources).toHaveBeenCalledWith(org, app, 'nb', { [id]: value + 
-additionalValue });
+    expect(queriesMock.upsertTextResources).toHaveBeenCalledWith(org, app, 'nb', {
+      [id]: value + additionalValue,
+    });
+  });
+
+  it('Check that reload is called when text is updated', async () => {
+    const reload = jest.fn();
+    const id = 'some-id';
+    const value = 'Lorem';
+    const resources: ITextResources = { nb: [{ id, value }] };
+    const previewIframeRefMock = createRef<HTMLIFrameElement>();
+    const previewIframeRef: RefObject<HTMLIFrameElement> = {
+      current: {
+        ...previewIframeRefMock.current,
+        contentWindow: {
+          ...previewIframeRefMock.current?.contentWindow,
+          location: {
+            ...previewIframeRefMock.current?.contentWindow?.location,
+            reload,
+          },
+        },
+      },
+    };
+    await render(resources, id, previewIframeRef);
+    expect(reload).toBeCalled;
```

Comment:
Husk parentes etter funksjonen. Uten parentes blir den ikke kalt, så her blir det faktisk ikke gjort noen sjekk.
```suggestion
    expect(reload).toBeCalled();
```
Jeg ville for øvrig også sjekket om den ikke blir kalt mer enn én gang:
```suggestion
    expect(reload).toHaveBeenCalledTimes(1);
```